pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -44,24 +44,64 @@ Quick link:
 
 ## Introduction {#intro}
 
-<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is a W3C Group Note that provides informative guidance (not normative requirements).
+<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is the acronym used for the W3C Guidance on Applying WCAG 2 to Non-web Information and Communication Technology Group Note.
+
+The WCAG2ICT Group Note provides:
+* [Informative](https://www.w3.org/TR/WCAG22/#dfn-informative) **guidance** that is not [normative](https://www.w3.org/TR/WCAG22/#dfn-normative) and that does not set requirements.
+* **Comments on applicability** of the Web Content Accessibility Guidelines (WCAG) when applied to non-web information and communications technologies (ICT) - this includes  non-web documents and software, including mobile apps, native applications, platform software, and [software on ICT with closed functionality[(https://www.w3.org/TR/wcag2ict/#comments-on-closed-functionality).
+* **Technology-specific explanations or difficulties** for some success criteria. This is especially the case where WCAG makes assumptions on the presence of a browser that would not be true for non-web documents and software. Not all ICT have these concepts.
```

Comment:
Explanations or difficulties? Both? One of those? Something else? I'd go with challenges.
"... true for non-web documents and software. Not all ICT have these concepts" seems to me like a redundant sentence. The first sentence already explains this concept.

```suggestion
* **Technology-specific challenges** for some success criteria. This is especially the case where WCAG makes assumptions on the presence of a browser that would not be true for non-Web documents and software.
```

Reply:
Sometimes we have explanations. Sometimes we have both. Maybe that sentence should instead read: 
"Technology-specific explanations or difficulties, as applicable, for the success criteria."

Again...same comment as before for using "non-Web" vs. "non-web".

Reply:
1) We talk to explanations or difficulties , thus don't want to incorporate the word change of challenges.
2) non-web is used as adjective, we don't want to accept these changes. Web is only capitalized when not an adjective.
3) Action item is to change Not all ICT have these concepts to Not all ICT have user agents for underlying platform (see Mary Jo's edits below)